### PR TITLE
docs: Align MSVC build options with Linux build ones 

### PR DESCRIPTION
**Pull request by hebasto**
Ref:
- #11526
- https://github.com/bitcoin/bitcoin/pull/15903#issuecomment-487139503
- https://github.com/bitcoin/bitcoin/pull/15903#issuecomment-487157470 by MarcoFalke

### CODE REVIEW

File: doc/release-process.md [hunk 1]
```diff
@@ -1,18 +1,26 @@
...
+  - set `CLIENT_VERSION_REVISION` to `0`
+  - set `CLIENT_VERSION_IS_RELEASE` to `true`
+
 Before every release candidate:
 
 * Update translations (ping wumpus on IRC) see [translation_process.md](https://github.com/bitcoin/bitcoin/blob/master/doc/translation_process.md#synchronising-translations).
-
 * Update manpages, see [gen-manpages.sh](https://github.com/bitcoin/bitcoin/blob/master/contrib/devtools/README.md#gen-manpagessh).
-* Update release candidate version in `configure.ac` (`CLIENT_VERSION_RC`)
+* Update release candidate version in `configure.ac` and `build_msvc/bitcoin_config.h` (`CLIENT_VERSION_RC`).
```
Comment (MarcoFalke) [inline doc/release-process.md]: msvc is a dev-only build and never released, the version stays at .99

File: doc/release-process.md [hunk 2]
```diff
@@ -1,18 +1,26 @@
...
 
 Before every minor and major release:
 
 * Update [bips.md](bips.md) to account for changes since the last release.
-* Update version in `configure.ac` (don't forget to set `CLIENT_VERSION_IS_RELEASE` to `true`) (don't forget to set `CLIENT_VERSION_RC` to `0`)
-* Write release notes (see below)
+* Update version in `configure.ac` and `build_msvc/bitcoin_config.h` (don't forget to set `CLIENT_VERSION_RC` to `0`).
```
Comment (MarcoFalke) [inline doc/release-process.md]: msvc is a dev-only build and never released, the version stays at .99

File: doc/release-process.md [hunk 3]
```diff
@@ -1,18 +1,26 @@
...
 ====================
 
+Just after branching off a new release:
+
+* In `configure.ac` and `build_msvc/bitcoin_config.h` on _the master branch_:
+  - update `CLIENT_VERSION_MINOR` version
+* In `configure.ac` and `build_msvc/bitcoin_config.h` on _a new release branch_:
+  - update `CLIENT_VERSION_MINOR` version
+  - set `CLIENT_VERSION_REVISION` to `0`
+  - set `CLIENT_VERSION_IS_RELEASE` to `true`
```
Comment (MarcoFalke) [inline doc/release-process.md]: Could link to an example commit 742f7dd972fca3dd4a33cfff90bf901b71a687e7?

Could move this down to section *Before a major release*?

**Comment by hebasto** [comment]
@MarcoFalke Thank you for your review. Your comments have been addressed.

**Comment by MarcoFalke** [comment]
utACK e47dc4f68bd3745d99dd31fe5b18b071b719c642

**Comment by Sjors** [comment]
utACK 47dc4f

**Comment by fanquake** [comment]
utACK e47dc4f

**Comment by practicalswift** [comment]
utACK e47dc4f68bd3745d99dd31fe5b18b071b719c642

**Pull request merged**